src/app/zap-templates/zcl/data-model/chip/energy-evse-cluster.xml

```diff
@@ -0,0 +1,221 @@
+    <cluster code="0x0099"/>
+    <item name="Disabled" value="0x00"/>
+    <item name="ChargingEnabled" value="0x01"/>
+    <item name="DischargingEnabled" value="0x02"/>
+    <item name="DisabledError" value="0x03"/>
+    <item name="DisabledDiagnostics" value="0x04"/>
+  </enum>
+  <enum name="FaultStateEnum" type="enum8" apiMaturity="provisional">
+    <cluster code="0x0099"/>
+    <item name="NoError" value="0x00"/>
+    <item name="MeterFailure" value="0x01"/>
+    <item name="OverVoltage" value="0x02"/>
+    <item name="UnderVoltage" value="0x03"/>
+    <item name="OverCurrent" value="0x04"/>
+    <item name="ContactWetFailure" value="0x05"/>
+    <item name="ContactDryFailure" value="0x06"/>
+    <item name="GroundFault" value="0x07"/>
+    <item name="PowerLoss" value="0x08"/>
+    <item name="PowerQuality" value="0x09"/>
+    <item name="PilotShortCircuit" value="0x0A"/>
+    <item name="EmergencyStop" value="0x0B"/>
+    <item name="EVDisconnected" value="0x0C"/>
+    <item name="WrongPowerSupply" value="0x0D"/>
+    <item name="LiveNeutralSwap" value="0x0E"/>
+    <item name="OverTemperature" value="0x0F"/>
+    <item name="Other" value="0xFF"/>
+  </enum>
+  <enum name="EnergyTransferStoppedReasonEnum" type="enum8" apiMaturity="provisional">
+    <cluster code="0x0099"/>
+    <item name="EVStopped" value="0x00"/>
+    <item name="EVSEStopped" value="0x01"/>
+    <item name="Other" value="0x02"/>
+  </enum>
+  <bitmap name="TargetDayOfWeekBitmap" type="bitmap8" apiMaturity="provisional">
+    <cluster code="0x0099"/>
+    <field name="Sunday" mask="0x01"/>
+    <field name="Monday" mask="0x02"/>
+    <field name="Tuesday" mask="0x04"/>
+    <field name="Wednesday" mask="0x08"/>
+    <field name="Thursday" mask="0x10"/>
+    <field name="Friday" mask="0x20"/>
+    <field name="Saturday" mask="0x40"/>
+  </bitmap>
+  <struct name="ChargingTargetStruct" apiMaturity="provisional">
+    <cluster code="0x0099"/>
+    <item fieldId="0" name="TargetTime" type="int16u" min="0" max="1439"/>
+    <item fieldId="1" name="TargetSoC" type="percent" optional="true"/>
+    <item fieldId="2" name="AddedEnergy" type="int64s" min="0" max="500000000" optional="true"/>
+  </struct>
+  <cluster apiMaturity="provisional">
+    <name>Energy EVSE</name>
+    <domain>Energy Management</domain>
+    <code>0x0099</code>
+    <define>ENERGY_EVSE_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>Electric Vehicle Supply Equipment (EVSE) is equipment used to charge an Electric Vehicle (EV) or Plug-In Hybrid Electric Vehicle. This cluster provides an interface to the functionality of Electric Vehicle Supply Equipment (EVSE) management.</description>
+    <!--Attributes-->
+    <attribute code="0x0000" side="server" type="StateEnum" define="STATE" isNullable="true" writable="false" optional="false">State</attribute>
```

Comment:
please add a `<globalAttribute side="server" code="0xFFFD" value="1" />` here to be explicit about version

Reply:
Have added it but set to ="2" since the spec update PR https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7813 set it to 2 after the changes in 0.7 ballot.